src/ssl.c

```diff
@@ -33401,6 +33405,11 @@ int wolfSSL_ECDSA_do_verify(const unsigned char *d, int dlen,
         WOLFSSL_MSG("wc_ecc_verify_hash incorrect signature detected");
         return WOLFSSL_FAILURE;
     }
+    #else
+        (void) dlen;
+        (void) check_sign;
+        return SSL_NOT_IMPLEMENTED;
```

Comment:
Instead of erroring out with `SSL_NOT_IMPLEMENTED`, it seems like for that case we could call `wc_ecc_verify_hash()`, which calls `wc_CryptoCb_EccVerify()` internally.  Looks like we would need to convert ECC {r,s} into a single signature first with `wc_ecc_rs_to_sig()`.

Reply:
Agree. Add to call `wc_ecc_rs_to_sig()` and `wc_ecc_verify_hash()`.
Also add unit test case if possible